services/frontend-v3/src/i18n/fr_CA.json

```
@@ -326,6 +327,7 @@
 
   "profile.visibility.show.confirm": "Etes-vous certain de vouloir rendre cette carte visible sur votre profil public?",
```

Probably no because of you, but could you add the accent on the "Êtes"
```suggestion
  "profile.visibility.show.confirm": "Êtes-vous certain de vouloir rendre cette carte visible sur votre profil public?",
```